src/neptune/attributes/namespace.py

```diff
@@ -99,6 +102,14 @@ def assign(self, value: Union[NamespaceVal, dict, Mapping], *, wait: bool = Fals
             value = NamespaceVal(value)
 
         for k, v in value.value.items():
+            if len(parse_path(k)) == 0:
```

Comment:
I'm not sure about this value verification here. Some of the types that we have (`Artifact`, `File`, and a few ones) does have a verification inside of a constructor (`__init__`). Could we move this directly to the constructor of `Namespace` as well? In addition, let's just filter out all the unwanted values and log a combined warning like `"", "/" could not be used as a name of an attribute`

Reply:
Done and done